packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -261,6 +262,11 @@ class Scrollable extends StatefulWidget {
   /// [ScrollBehavior].
   final ScrollBehavior? scrollBehavior;
 
+  /// {@macro flutter.material.Material.clipBehavior}
+  ///
+  /// Defaults to [Clip.hardEdge].
+  final Clip clipBehavior;
```

Comment:
In this particular case, we should probably document that it does not apply clipping to the Scrollable, rather that is is passed on to decorators in ScrollableDetails.

Reply:
That's a great idea, this is not a regular `clipBehavior` property and I tried to find a similar use case of this property in other widgets yesterday and I couldn't find anything like this so I kept it as it is. Updating this doc.